versions.tf

```diff
@@ -20,7 +20,7 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 6.9.0, < 7"
+      version = ">= 6.37.0, < 7"
```

Comment:
I assume 6.37 minimum version is required for this feature. Shouldn't `modules/simple_bucket/versions.tf` be also updated?